client/spam_evader.py

```diff
@@ -0,0 +1,115 @@
+	"standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those interested. Sections 1.10.32 and 1.10.33 from 'de Finibus Bonorum et " \
+	"Malorum' by Cicero are also reproduced in their exact original form, accompanied by English versions from the 1914 translation by H. Rackham. The " \
+	"hilarious fact of all of this is that the reader has absolutely no idea this text even exists, and that it is simply a placeholder designed to subvert " \
+	"security controls on mail filters. Because mail filters are so primitive, they are unable to distinguish valid text from placeholder text, especially in " \
+	"the context of a social engineering email. Since these sentences are not using random generated text, they will inevitably be used in an attempt to bypass" \
+	" security controls designed to catch words which frequently appear in illegitimate emails. But why am I even telling you this? You'll never even get to see it." \
+	" If you did, bravo, you have seen through the ruse or simply have html formatting turned off, in which case you got lucky. Oh well. You can't win them all," \
+	" I suppose. From Russia, with love.</p>"
+
+
+try:
+	import markovify
+except ImportError:
+	has_markov = False
+else:
+	has_markov = True
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Spencer McIntyre', 'Mike Stringer']
+	title = 'Spam Evader'
+	description = """
+	Add and modify custom html messages from a file to reduce spam assassin 
+	scores. This plugin interacts with the message content to append a long
+	series of randomly generated sentences to meet the ideal image-text ratio.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+	options = [
+		plugins.ClientOptionString(
+			'corpus',
+			description='text file containing text to generate dynamic padding',
+			default=os.path.join(PLUGIN_PATH, 'corpus.txt'),
+			display_name='Corpus File'
+		),
+
+		plugins.ClientOptionString(
+			'dynamic_padding',
+			description='Sets whether dynamic generated or static padding is appended to the messaged',
+			default=True
+		)
+	]
+	req_min_version = _min_version
+	version = '1.0.1'
+	req_packages = {
+		'python3-markovify': has_markov
+	}
+
+	spam_words = {}
+
+	def initialize(self):
+		mailer_tab = self.application.main_tabs['mailer']
+		self.signal_connect('message-create', self.signal_message_create, gobject=mailer_tab)
+		if os.path.isfile(os.path.realpath(self.config['corpus'])):
+			self.corpus = os.path.realpath(self.config['corpus'])
+		else:
+			self.corpus = False
+		self.logger.debug("Spam Evader: Plugin path \t " + PLUGIN_PATH)
+		self.logger.debug("Spam Evader: Corpus File \t" + self.corpus)
+		if self.corpus and has_markov:
+			self.dynamic = self.config['dynamic_padding']
+		else:
+			self.dynamic = False
+
+		return True
+
+	def signal_message_create(self, mailer_tab, target, message):
+		payload = [part for part in message.walk()]
```

Comment:
I will make it so that it only modifies text/html content for the time being. If we do decide to implement support for text/plain messages as well, this part can be easily modified to use the startswith prefix:
Ex:
`part.get_content_type().startswith(prefix=('text/html', 'text/plain'))`